packages/@sanity/desk-tool/src/DeskTool.test.tsx

```diff
@@ -0,0 +1,549 @@
+/* eslint-disable max-nested-callbacks */
+/// <reference types="@sanity/types/parts" />
+
+import {render, act} from '@testing-library/react'
+import React, {useEffect, useMemo, useState} from 'react'
+import {Subject, of} from 'rxjs'
+import {useRouter} from '@sanity/base/router'
+import * as ops from 'rxjs/operators'
```

Comment:
wasn't clear to me when reading the code in this file that `ops` was all the rxjs operators. By established convention elsewhere we import the individual operators as we need them. Any reason to not do that here? (imo it would also be more readable if it was named `operators` instead of `ops`)

Reply:
In the test file i found it pretty annoying to write imports for the different operators while deving and these test files won't ever be shipped to users so we don't have worry about tree shaking etc. maybe i just need to get better tooling for auto imports haha